src/routes/tools/wildfire/_data.js

```diff
@@ -142,25 +142,27 @@ export function getQueryParams({
   simulation,
   monthNumber,
   imperial = false,
+  climvar,
 }) {
   const params = {
     imperial,
+    stat: climvar === "fire" ? "sum" : "mean",
+    // countnd: true,
     ...(simulation === "month" && { months: monthNumber }),
   };
   let method;
   switch (boundary.id) {
     case "locagrid":
       params.g = `Point(${location.center[0]} ${location.center[1]})`;
+      params.stat = "mean";
```

Comment:
you dont need the `stat` param for the locagrid. although it doesn't look like it changes the values.

Reply:
Got it, I guess I could delete it here instead